src/components/picker/helpers/usePickerSelection.tsx

```diff
@@ -13,6 +13,13 @@ const usePickerSelection = (props: UsePickerSelectionProps) => {
   const [multiDraftValue, setMultiDraftValue] = useState(value as PickerMultiValue);
   const [multiFinalValue, setMultiFinalValue] = useState(value as PickerMultiValue);
 
+  useEffect(() => {
+    if (multiFinalValue !== value) {
```

Comment:
Do we always want to do this or just when the picker is in "multi" mode? 
Could this cause an issue in the "single" mode?

Reply:
I think it should not cause an issue, but it's probably more readable and will cause less renders 👍